back/engines/commercial/user_custom_fields/spec/acceptance/user_custom_fields_spec.rb

```diff
@@ -254,24 +267,31 @@
     end
 
```

Comment:
Updated the following test to be sure that when a custom field is deleted, the values and the permissions relationships are also deleted.